src/Common/src/TypeSystem/IL/ILImporter.cs

```diff
@@ -304,13 +313,16 @@ private void MarkBasicBlock(BasicBlock basicBlock)
 
         private void ImportBasicBlock(BasicBlock basicBlock)
         {
-            _stackTop = 0;
+            _stack.Clear();
```

Comment:
I thought about this too but was surprised that the definition and usage were divided among those 2 files.